src/modules/static/components/footer.jsx

```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
 	render() {
 		return (
 			<footer>
-
 				<div className="imprint">
-					<div className="container-fluid">
-						<div className="right">
-						<a href="https://hpi.de/impressum.html" target="blank"><i className="fa fa-info" aria-hidden="true"></i> Impressum</a><br/>
-						<a href="mailto:hpi-info@hpi.de?subject=Schul_Cloud%20Anfrage"><i className="fa fa-at" aria-hidden="true"></i> Kontakt</a>
-						</div>
-						<div className="bg-graphic bg-graphic-footer"></div>
-					</div>
 					<div className="center">
 						<img src="https://hpi.de/fileadmin/templates/img/hpi_header_logo_pos_srgb.png"></img>
```

Comment:
Don't use static images from other servers as their paths could change. Please add the image to this repo directly and use relativ paths.